### PR TITLE
Dropseq

### PR DESCRIPTION
**Pull request by gsmashd**

### CODE REVIEW

File: single-cell/rules/quant/star.rules [hunk 1]
```diff
@@ -74,7 +74,7 @@ rule starsolo_quant:
         umi_start = config['quant']['starsolo']['umi_start'],
         R1 = starsolo_R1,
         R2 = starsolo_R2,
-        extra_args = '--genomeLoad LoadAndKeep --soloFeatures Gene GeneFull SJ Velocyto Transcript3p ' 
+        extra_args = '--genomeLoad LoadAndKeep --outFilterMultimapNmax 1 --soloFeatures Gene GeneFull SJ Velocyto Transcript3p '
```
Comment (flatberg) [inline single-cell/rules/quant/star.rules:77]: insert option for multimapper output in sam file?

File: single-cell/rules/quant/star.rules [hunk 2]
```diff
@@ -35,6 +35,19 @@ rule starsolo_genome_index:
...
+    input:
+        join(UMI_INTERIM, '{sample}','whitelist.txt')
+    output:
+        join(STAR_INTERIM, '{sample}', 'whitelist.txt')
+    threads:
+        1
+    singularity:
+        'docker://' + config['docker']['default']
+    shell:
+        'awk -F\"\\t\" \'{{print $1}}\' {input} > {output} '
```
Comment (flatberg) [inline single-cell/rules/quant/star.rules:48]: use tripple quotes ?

File: single-cell/rules/bfq/bfq_level2.rules [hunk 3]
```diff
@@ -50,7 +50,7 @@ rule bfq_level2_aligned:
 
 rule bfq_level2_umap_png:
     input:
-        join(QUANT_INTERIM, 'aggregate', '{quant}', '{aggr_id}_aggr.h5ad')
+        join(QUANT_INTERIM, 'aggregate', '{quant}', 'scanpy', '{aggr_id}_aggr.h5ad')
```
Comment (flatberg) [inline single-cell/rules/bfq/bfq_level2.rules:53]: join(QUANT_INTERIM, 'aggregate', '{quant}', 'merged_aggr.h5ad') ?

File: single-cell/rules/quant/star.rules (hunk 1 above)
Comment (gsmashd) [reply single-cell/rules/quant/star.rules:79]: --outSAMmultNmax follows --outFilterMultimapNmax by default

**Pull request merged**